crates/common/consensus/lean/src/vote.rs

```diff
@@ -12,7 +12,7 @@ use crate::checkpoint::Checkpoint;
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash)]
 pub struct SignedVote {
     pub data: Vote,
-    pub signature: PQSignature,
+    pub signature: FixedBytes<4000>,
 }
 
```

Comment:
```suggestion
#[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize, Encode, Decode, TreeHash)]
pub struct SignedVote {
    pub validator_id: uint64
    pub message: Vote,
    pub signature: FixedBytes<4000>,
}

```

Refer to `SignedVote` [definition](https://github.com/leanEthereum/leanSpec/blob/main/docs/client/containers.md#signedvote).

Reply:
Also note that we should delete `validator_id` in `Vote`.